card/mod/05_standard/set/all/event_viz.rb

```diff
@@ -25,45 +26,42 @@ def puts_events events, prefix='', depth=0
   events.each do |e|
     space = ' ' * (depth * 2)
 
-    #FIXME - this is not right.  before and around callbacks are processed in declaration order regardless of kind.
-    # not all befores then all arounds
+    # FIXME: this is not right.  before and around callbacks are processed in
+    # declaration order regardless of kind.  not all befores then all arounds
 
     if e[:before]
-      r += puts_events( e[:before], space+'v  ', depth)
+      r += puts_events(e[:before], space+'v  ', depth)
     end
     if e[:around]
-      r += puts_events( e[:around], space+'vv ', depth )
+      r += puts_events(e[:around], space+'vv ', depth)
     end
 
-
     output = "#{prefix}#{e[:name]}"
-    #warn output
+    # warn output
     r+= "#{output}\n"
 
     if e[:after]
-      r += puts_events( e[:after ].reverse, space+'^  ', depth )
+      r += puts_events(e[:after ].reverse, space+'^  ', depth)
```

Comment:
Surrounding space missing for operator `+`.<br>Space inside square brackets detected.